src/dlgPackageExporter.cpp

```diff
@@ -866,13 +866,9 @@ void dlgPackageExporter::writeConfigFile(const QString& stagingDirName, const QF
     appendToDetails(qsl("description"), packageDescription);
     appendToDetails(qsl("version"), ui->lineEdit_version->text());
     appendToDetails(qsl("dependencies"), dependencies.join(","));
-    QDateTime iso8601timestamp = QDateTime::currentDateTime();
-    const int offset = iso8601timestamp.offsetFromUtc();
-    iso8601timestamp.setOffsetFromUtc(offset);
-    QDateTime iso8601time(QDateTime::currentDateTime());
-    iso8601time.setTimeSpec(Qt::OffsetFromUTC);
```

Comment:
These two lines are actually cruft - they have no effect as the variable `iso8601time` is not used!